src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -197,10 +248,22 @@ public JedisCluster(ClusterConnectionProvider provider, int maxAttempts,
     super(provider, maxAttempts, maxTotalRetriesDuration);
   }
 
+  /**
+   * Returns all nodes that were configured to connect to in a KEY-VALUE pair (Map).
+   * Key is the HOST:PORT and the value is the connection.
+   * @return Map of all connections.
+   */
   public Map<String, ConnectionPool> getClusterNodes() {
     return ((ClusterConnectionProvider) provider).getNodes();
   }
 
+  /**
+   * Returns the connection for one of the 16,384 slots.<br><br>
+   * If there is no connection for the given slot, either cause the node died or didn't connect, it may not return the correct node.
+   * It will try to resolve connection-based issues using reconnection.
```

Comment:
Don't want to give much explanation to this. This is not a recommended method use. It is there in case for things that are not available in JedisCluster.
```suggestion
   * Returns the connection for one of the 16,384 slots.
```